### PR TITLE
Fix: Fixed an issue where the Properties window had the wrong icon

### PR DESCRIPTION
**Pull request by 0x5bfa**
## Summary

As the title.

## Task Checklist

_N/A_

## Steps To Validate Changes

1. Launch the app
2. Open a properties window
3. See window icon from the taskbar

## Known Issues

_N/A_

## PR Checklist

- [ ] **Development**: _N/A_
- [ ] **Approval**: Have discussed with and got approval from the team[^1]
- [x] **Tests**: Tested accessibility on the local end
- [x] **Deployment**: Deployed the app on the local end
- [ ] **Localization**: Modified en-US string resources[^2]
- [ ] **Co-authors**: _N/A_

## Screenshots

Before:
image

[^1]: If the request hasn't been agreed by the team, this work might be rejected. Make sure that you get approval from the team before opening any PR related the request.
[^2]: If you removed any en-US string resources, make sure that there are no references of those resources. When you add a new en-US string resources, do not make any changes on other languages' resources; [Crowdin](https://crowdin.com/project/files-app) will do that, instead.

### CODE REVIEW

File: src/Files.App/Utils/Storage/Helpers/FilePropertiesHelpers.cs [hunk 1]
```diff
@@ -120,7 +121,14 @@ public static void OpenPropertiesWindow(object item, IShellPage associatedInstan
 			appWindow.TitleBar.ButtonBackgroundColor = Colors.Transparent;
 			appWindow.TitleBar.ButtonInactiveBackgroundColor = Colors.Transparent;
 
-			appWindow.SetIcon(applicationService.AppIcoPath);
+			string appIcoPath = ApplicationService.AppEnvironment switch
```
Comment (yaira2) [inline src/Files.App/Utils/Storage/Helpers/FilePropertiesHelpers.cs:124]: Doesn't the application service already contain this logic?

Comment (0x5bfa) [reply src/Files.App/Utils/Storage/Helpers/FilePropertiesHelpers.cs:124]: That was marked as obsolete.

Comment (yaira2) [reply src/Files.App/Utils/Storage/Helpers/FilePropertiesHelpers.cs:124]: The service or just that method?

Comment (0x5bfa) [reply src/Files.App/Utils/Storage/Helpers/FilePropertiesHelpers.cs:124]: Just that.

Comment (yaira2) [reply src/Files.App/Utils/Storage/Helpers/FilePropertiesHelpers.cs:124]: I think it's worth having it in a single place instead of duplicating it.

Comment (0x5bfa) [reply src/Files.App/Utils/Storage/Helpers/FilePropertiesHelpers.cs:124]: Should I do that in this PR?

Comment (yaira2) [reply src/Files.App/Utils/Storage/Helpers/FilePropertiesHelpers.cs:124]: Might as well 👍

Comment (0x5bfa) [reply src/Files.App/Utils/Storage/Helpers/FilePropertiesHelpers.cs]: I decided to use AppLifecycleHelper to put those values. Thanks for your idea.

File: src/Files.App/Helpers/Application/AppLifecycleHelper.cs [hunk 2]
```diff
@@ -26,6 +27,39 @@ namespace Files.App.Helpers
...
+		public static Version AppVersion { get; } = new(
+			Package.Current.Id.Version.Major,
+			Package.Current.Id.Version.Minor,
+			Package.Current.Id.Version.Build,
+			Package.Current.Id.Version.Revision);
+
+		/// <summary>
+		/// Gets application icon relative path.
+		/// </summary>
+		public static string AppIconPath { get; } = AppEnvironment switch
```
Comment (yaira2) [inline src/Files.App/Helpers/Application/AppLifecycleHelper.cs:56]: What do you think about combining this with `Package.Current.InstalledLocation.Path`?

Comment (0x5bfa) [reply src/Files.App/Helpers/Application/AppLifecycleHelper.cs:56]: Fair enough, thanks!

**Comment by 0x5bfa** [comment]
@yaira2 I think we better use MicaAlt and background mask to be consistent to the main window coloring. How about?
image

File: src/Files.App/Helpers/Application/AppLifecycleHelper.cs (hunk 2 above)
Comment (0x5bfa) [reply src/Files.App/Helpers/Application/AppLifecycleHelper.cs:56]: Done

**Comment by yaira2** [comment]
MicaAlt is best used when there are more than two layers

**Review by yaira2 — APPROVED** [review]

**Pull request merged**